### PR TITLE
	RavenDB-7818 Remove bad implementation to compare change vectors

### PR DESCRIPTION
**Pull request by fitzchak**

### CODE REVIEW

File: src/Raven.Server/Documents/Replication/OutgoingReplicationHandler.cs [hunk 1]
```diff
@@ -415,7 +411,7 @@ private class UpdateSiblingCurrentEtag : TransactionOperationsMerger.MergedTrans
         {
             private readonly DocumentDatabase _database;
             private readonly ReplicationMessageReply _replicationBatchReply;
-            private  Guid _dbid;
+            private  Guid _dbDd;
```
Comment (ayende) [inline src/Raven.Server/Documents/Replication/OutgoingReplicationHandler.cs:414]: Huh? Why this change?

Comment (fitzchak) [reply src/Raven.Server/Documents/Replication/OutgoingReplicationHandler.cs:414]: Sorry, I meant to change it to _dbId. Fixed.
